scalahost/sbt/src/main/scala/scala/meta/scalahost/sbt/ScalahostSbtPlugin.scala

```diff
@@ -60,10 +60,7 @@ object ScalahostSbtPlugin extends AutoPlugin {
     javaOptions ++= {
       if (scalametaDependencies.value.isEmpty) Nil
       else {
-        val sourcepath =
-          scalahostSourcepath.value
-            .flatMap(_.map(_.getAbsolutePath))
-            .mkString(java.io.File.pathSeparator)
+        val sourcepath = sys.props("user.dir")
```

Comment:
We can definitely improve the implementations on deep and find if the need arises. At the moment, they are intentionally very naive in order to focus the development effort.